src/04-arrays.js

```diff
@@ -135,7 +155,10 @@ function removeElementFromFrontOfArray(array) {}
  *  getMiddleElement([ 10, null, "30" ]);
  *  //> null
  */
-function getMiddleElement(array) {}
+function getMiddleElement(array) {
+  let midEl = Math.floor(array.length / 2);
+  return array [midEl]
```

Comment:
I think we could find a better varaible name -` mid ` or ` middleEl ` or ` midElement ` seems to work a little better.  If we have to abbreviations in one variable name it can start to lose meaning.  Also lets connect ` array ` and `[midEl]`